wire-ios/Wire-iOS/Sources/Authentication/Interface/Views/ValidatedTextField.swift

```diff
@@ -315,6 +315,10 @@ final class ValidatedTextField: AccessoryTextField, TextContainer, Themeable {
         return enableConfirmButton?() ?? !input.isEmpty
     }
 
+    var isValid: Bool {
```

Comment:
I know the name of that property is not that good. Any suggestions are more than appreciated!

Reply:
do you still use it?

Reply:
Yes!